Jenkinsfile

```diff
@@ -48,18 +48,18 @@ pipeline {
     options {
         buildDiscarder logRotator(numToKeepStr: '15')
     }
-    
+
     environment {
         YARN_CACHE_FOLDER = "${env.WORKSPACE}/yarn-cache"
         SPAWN_WRAP_SHIM_ROOT = "${env.WORKSPACE}"
+        EMAIL_TO = "ndoschek+eclipseci@eclipsesource.com, eneufeld+modelserver-ci@eclipsesource.com"
```

Comment:
why not use same name after the `+` ?

Reply:
I'll update your email addresses to the same suffix as mine